dialog/en-us/wiki.specific.response.dialog

```diff
@@ -0,0 +1 @@
+Here is your answer from wiki peedia
```

Comment:
Should be wikipedia, if this is done to improve the Mimic2 speech I think it should be done in the backend word improvement system.

Reply:
i think this should go here https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/res/text/en-us/phonetic_spellings.txt

Reply:
Jarbas, this is interesting. How is this currently used exactly? First time I have seen it. I mean I agree with you guys I just don't quite understand the specifics.

Reply:
its loaded in the tts class https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/tts/tts.py#L196 and used to replace strings in an utterance in the synth step https://github.com/MycroftAI/mycroft-core/blob/dev/mycroft/tts/tts.py#L347